modules/vm/main.tf

```diff
@@ -0,0 +1,187 @@
+data "azurerm_resource_group" "rg" {
```

Comment:
These can be created with Terraform but let's do that in another MR.

Reply:
If we drop the RG, we pretty much have to do the network, subnets, etc. as well. Pretty much all data needs to go to resources. It can be a different MR.